### PR TITLE
Lm animation fix other browser

### PR DESCRIPTION
**Pull request by LarryMatte**

### CODE REVIEW

File: src/main/resources/com/arcbees/website/client/resources/css/animations.gss [hunk 1]
```diff
@@ -6,17 +6,53 @@
...
-    100% {transform:rotate(90deg) translate(160px, 0px); opacity: 0;}
+    0% {
+        -moz-transform:rotate(0deg);
+        -ms-transform:rotate(0deg);
+        -o-transform:rotate(0deg);
+        transform:rotate(0deg);
+        opacity: 1;
+    }
+    25% {
+        -moz-transform:rotate(90deg);
```
Comment (LarryMatte) [inline src/main/resources/com/arcbees/website/client/resources/css/animations.gss]: for some reasons, mixins won't work...

File: src/main/resources/com/arcbees/website/client/resources/css/developers.gss [hunk 2]
```diff
@@ -158,7 +158,7 @@ ul.social {
 .slideshow a.prev::before,
 .slideshow a.next::before {
     content: "\e631";
-    font-family: "icons";
+    font-family: "icomoonregular";
```
Comment (LarryMatte) [inline src/main/resources/com/arcbees/website/client/resources/css/developers.gss]: had to change it to make it work in firefox... we'll find a way to rename it somehow

File: src/main/resources/com/arcbees/website/client/resources/css/animations.gss (hunk 1 above)
Comment (christiangoudreau) [inline src/main/resources/com/arcbees/website/client/resources/css/animations.gss]: @jDramaix

**Comment by jasonlemay** [comment]
LGTM

**Pull request merged**